### PR TITLE
removing previous starter code and adding NextJS/TailwindCSS code

### PR DESCRIPTION
**Pull request by wsedor**
# Description
Changes here are: 
- To remove the starter code previously utilized for the web app as it was causing unnecessary package errors and development issues. 
- Add in NextJS and TailwindCSS starter code setup using typescript from the beginning. Followed steps here https://tailwindcss.com/docs/guides/nextjs
- Changes are for Asana task https://app.asana.com/0/1204100804247509/1204314026899292/f

# Testing
- Frontend running correctly
Screenshot 2023-04-10 194336

- Homepage view
homepage

# Reviewers
@AmoabaKelvin
@KomlaRD

### CODE REVIEW

File: webapp/README.md [hunk 1]
```diff
@@ -1,39 +1,20 @@
-## NSMQ AI Web application
```
Comment (wsedor) [inline webapp/README.md:1]: why was this removed?

Comment (wsedor) [inline webapp/README.md:1]: why was this removed?

Comment (wsedor) [inline webapp/README.md:1]: why was this removed?

Comment (wsedor) [inline webapp/README.md:1]: 123

File: webapp/frontend/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```
Comment (KomlaRD) [inline webapp/frontend/tsconfig.json:3]: Can we set target to ES6?

The following link provides comparison between ES5 and ES6
https://www.javatpoint.com/es5-vs-es6#:~:text=Difference%20between%20ES5%20and%20ES6&text=ES5%20supports%20primitive%20data%20types,symbol'%20for%20supporting%20unique%20values.

https://www.geeksforgeeks.org/difference-between-es5-and-es6/

**Review by KomlaRD — APPROVED** [review]

File: webapp/frontend/tsconfig.json (hunk 2 above)
Comment (wsedor) [reply webapp/frontend/tsconfig.json:3]: agreed. I missed to make the change here
